modules/ROOT/partials/konflux-additional-getting-started.adoc

```diff
@@ -0,0 +1,25 @@
+== {ProductName} personas
+
+{ProductName} is a platform that helps teams develop applications. How you use {ProductName} depends on what role you have on your team. Konflux users generally fall into one of two categories:
+
+* Platform Engineers (PEs): These are the people whose job isn't to engineer applications, but to make it possible for others to do so. They usually own the instance of {ProductName}. They manage that instance and its users.
+* Developers:  These are the people who engineer applications for their organization. They are the typical users of {ProductName}, who productize their software with it.
+
+Depending on your role, read whichever following section applies best to you, to get more information and links to further documentation.
+
+=== PEs
```

Comment:
Nitpick: I don't think the acronym here is helping in any way. Let's just write this as Platform Engineers.

Reply:
Fixing in a pending commit.